pkg/controllers/clusterresourceplacement/resource_selector.go

```diff
@@ -56,10 +57,18 @@ func (r *Reconciler) selectResources(ctx context.Context, placement *fleetv1alph
 	return manifests, nil
 }
 
+func convertResourceSelector(old []fleetv1alpha1.ClusterResourceSelector) []fleetv1beta1.ClusterResourceSelector {
```

Comment:
Just to make sure I understand the purpose correctly, this is for temporarily adding a wrapper layer to satisfy the tests? It will be removed when the resource conversion webhook is ready?

Reply:
are we converting CRD between api groups?

Reply:
nope, just temporary solution in order to not break the existing tests & logic.

i don't think we'll have the resource conversation webhook as we changed the group version.